docs/version0.33.md

```diff
@@ -27,6 +27,7 @@
 The following new features and notable changes since version 0.30.0 are included in this release:
 
 - [New binaries and changes to supported environments](#binaries-and-supported-environments)
+- [JITServer AOT caching and performance metrics](#jitserver-aot-caching-and-performance-metrics)
```

Comment:
I was wondering is JITServer support for OpenSSL 3.0 is something of interest to be added here.